lms/static/sass/views/_login-register.scss

```diff
@@ -350,6 +350,8 @@ $sm-btn-linkedin: #0077b5;
         @extend %btn-secondary-grey-outline;
         width: 130px;
         padding: 0 0 0 ($baseline*2);
+        margin-right: 6px;
+        margin-bottom: $baseline;
         height: 34px;
```

Comment:
Do you need to specify a height here? If not, I'd prefer this removed and the element's height be automatically calculated from its contents. If you need to sync up heights across buttons, I'd recommend either ``min-height``.